CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+# Enso 2.0.0-alpha.5 (2021-05-14)
+
+<br/>![New Features](/docs/assets/tags/new_features.svg)
+
+#### Visual Environment
+
+#### EnsoGL (rendering engine)
+
+<br/>![Bug Fixes](/docs/assets/tags/bug_fixes.svg)
+
+#### Visual Environment
+
+- [Delete key will delete selected nodes][1538].
+
+#### EnsoGL (rendering engine)
+
+#### Enso Compiler
+
+- [Updated Enso engine to version 0.2.11][1541].
+
+If you're interested in the enhancements and fixes made to the Enso compiler,
+you can find their release notes
+[here](https://github.com/enso-org/enso/blob/main/RELEASES.md).
+
+[1541]: https://github.com/enso-org/ide/pull/1511
+[1538]: https://github.com/enso-org/ide/pull/1538
```

Comment:
Also, this PR seems bigger than just "adding delete key" - but it is not described in changelog. Probably its OK, but Id love to learn more of why :)

Reply:
This is the change summary I see in delete PR, it seems reasonable:
![image](https://user-images.githubusercontent.com/3919101/120449745-00240680-c390-11eb-9113-6a31458342e0.png)

Are you sure you looked at the proper PR?